intelligence/tasks.py

```
@@ -92,4 +93,149 @@
+def run_daily_pattern_analysis():
+    """
+    تحلیل روزانه الگوهای تمام بیماران فعال
+    """
+    try:
+        from gitdm.models import PatientProfile
+        
+        # دریافت بیماران فعال (که در 3 ماه گذشته داده داشته‌اند)
+        cutoff_date = timezone.now() - timezone.timedelta(days=90)
+        
+        active_patients = PatientProfile.objects.filter(
+            Q(labresult__taken_at__gte=cutoff_date) |
```

Ruff (0.12.2)

Undefined name `Q`

(F821)